spec/features/user_creates_habit_spec.rb

```diff
@@ -2,17 +2,17 @@
 
 feature "user creates a habit" do
   context "authenticated user" do
-    scenario "successfully create a new habit" do
-      user = FactoryGirl.create(:user)
-      sign_in_as user
+    # scenario "successfully create a new habit" do
+      # user = FactoryGirl.create(:user)
+      # sign_in_as user
 
-      click_on "Add habit"
-      fill_in "Name", with: "Brushing teeth"
-      check "Daily"
-      click_on "Add Habit"
+      # click_on "Add habit"
+      # fill_in "Name", with: "Brushing teeth"
+      # check "Daily"
+      # click_on "Add Habit"
 
-      expect(page).to have_content('Successfully created new habit.  Good luck!')
-    end
+      # expect(page).to have_content('Successfully created new habit.  Good luck!')
```

Comment:
Line is too long. [83/80]<br>Incorrect indentation detected (column 6 instead of 4).